### DISCUSSION TITLE
Bottom statusline is unreadable.

### DISCUSSION THREAD

**Discussion by willdoescode** [arcticicestudio/nord#214] [Help & Support]
The bottom status line is way too bright to be able to read.
System: macOs
Vim: Spacevim

![Screen Shot 2020-10-30 at 00 12 18](https://user-images.githubusercontent.com/53620720/97670969-93850a00-1a44-11eb-8e83-1600fcf3a5b5.png)

**Comment by arcticicestudio** [comment]
Hi @willdoescode :wave:

In order to ensure the problem is related to the Nord theme you need to reproduce it with a minimal and clean configuration. This can be done by creating new configuration file with any name (e.g. `testrc`) and run Vim with the following command: `vim -u testrc`
For me as a theme author it is not possible to know if the theme is the root cause of the problem when there are other attributes set in the configuration because each one might break the theme.

Here's an minimal test configuration you can use to try to reproduce the problem (supposed you're using [vim-plug][]):

```vim
call plug#begin(expand('~/.vim/plugged'))
Plug 'arcticicestudio/nord-vim'
call plug#end()

set nocompatible
if (has("termguicolors"))
  set termguicolors
endif

syntax enable
colorscheme nord
```

As you can see in the example configuration I've added the `termguicolors` settings which could be one reason why the color is not rendered correctly. This is because the [`*_term` attributes][gh-blob-term_attr] make use of the colors that are provided by the terminal color scheme which might be for example [iTerm2][ref-iterm] or [Terminal.app][ref-term.app] in your case. The [`*_gui` variables][gh-blob-gui_attr] on the other side are used when Vim runs in GUI mode. So in order to run Vim in a terminal but use colors like in GUI mode the `termguicolors` variable must be set and your terminal emulator must support “true colors“ (24bit).

Unfortunately terminal colors are quite complex due to their long history, but modern terminal emulators should be all capable of rendering "True Colors". You should make sure to check out the great [“Terminal Colors“ gist][gist-tc] that provides many details about color rendering in terminal emulators, tracking of supported terminal emulator projects and answers to common confusions about terminal colors. The [“Colours in terminal“ gist][gist-cit] is also quite helpful to learn more about the actual rendering engines and standards used by these terminal emulators.
There are also a lot of blog posts like [this][bp1] and [this][bp2] one that help to handle different use case with various terminal emulators in combination with different CLI and TUI applications.

Also the colors on your screenshot are not even close to the actual style of Nord so I guess there is a general setup problem.
Could you please try to either use the example configuration or set the `termguicolors` setting in your current configuration and check whether colors are then displayed correctly?

[ref-term.app]: https://github.com/arcticicestudio/nord-terminal-app/blob/master/src/xml/Nord.terminal#L23-L31
[ref-iterm]: https://github.com/arcticicestudio/nord-iterm2/blob/master/src/xml/Nord.itermcolors#L187-L199
[vim-plug]: https://github.com/junegunn/vim-plug
[bp1]: https://sts10.github.io/2016/06/15/true-hex-colors-in-vim-with-iterm2.html
[bp2]: https://tomlankhorst.nl/iterm-tmux-vim-true-color
[gist-tc]: https://gist.github.com/XVilka/8346728
[gist-cit]: https://gist.github.com/sindresorhus/bed863fb8bedf023b833c88c322e44f9
[gh-blob-term_attr]: https://github.com/arcticicestudio/nord-vim/blob/ea7ff9c3/colors/nord.vim#L37-L49
[gh-blob-gui_attr]: https://github.com/arcticicestudio/nord-vim/blob/ea7ff9c3/colors/nord.vim#L19-L35

**Comment by jvoisin** [comment]
@willdoescode ping :)

**Comment by svengreb** [comment]
@willdoescode Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
